src/EPR.Calculator.API.Data/DataModels/CalculatorRun.cs

```diff
@@ -19,12 +19,12 @@ public class CalculatorRun
         [Column("name")]
         [StringLength(250)]
         [Required]
-        public string Name { get; set; }
+        public string? Name { get; set; }
 
         [Column("financial_year")]
         [StringLength(250)]
         [Required]
-        public string Financial_Year { get; set; }
+        public string? Financial_Year { get; set; }
```

Comment:
We cannot have Required and Optional?

Reply:
Fixed by replacing the nullable modifier with required, and adjusting unit tests to match.